winfsinjector/application.go

```diff
@@ -71,7 +70,7 @@ func (a Application) Run(inputTile, outputTile, registry, workingDir string) err
 
 	embeddedReleaseDir := filepath.Join(extractedTileDir, "embed/windowsfs-release")
 	if _, err := os.Stat(embeddedReleaseDir); os.IsNotExist(err) {
-		fmt.Println("The file system has already been injected in the tile; skipping injection")
+		fmt.Println("No file system found or the file system has already been injected in the tile; skipping injection")
```

Comment:
I think it would be cool if these cases were detected and logged separately. This is good as is but if you can easily distinguish the two, debugging would be easier.